spec/defines/windows_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'spec_helper'
+describe 'local_user::windows', :type => :define do
+  let (:title) { 'rnelson0' }
+  let (:params) do
+    {
+      :state    => 'present',
+      :groups   => [],
+      :password => 'Microsoft1',
+    }
+  end
+
+  context 'rdp user, no admin' do
+    it { is_expected.to create_user('rnelson0').with({
+      :comment          => 'rnelson0',
+      :groups           => ['Remote Desktop Users'],
+      :password         => 'Microsoft1'
+    }) }
+  end
+  
+  context 'local user only' do
```

Comment:
'with allow_rdp false' is a better description. The other contexts need similar changes, "with <param> <value>" is good.